lib/sidekiq/cli.rb

```
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
```

No need to collect all errors, just fail fast on the first error.